sagemaker-neo-notebooks/edge/raspberry-pi/neo-rasb3b-object-detection.ipynb

```diff
@@ -0,0 +1,1157 @@
+      "Requirement already satisfied: opt-einsum>=2.3.2 in /home/pi/.local/lib/python3.7/site-packages (from tensorflow) (3.1.0)\n",
+      "Requirement already satisfied: certifi>=2020.06.20 in /home/pi/.local/lib/python3.7/site-packages (from matplotlib) (2020.6.20)\n",
+      "Requirement already satisfied: kiwisolver>=1.0.1 in /home/pi/.local/lib/python3.7/site-packages (from matplotlib) (1.2.0)\n",
+      "Requirement already satisfied: cycler>=0.10 in /home/pi/.local/lib/python3.7/site-packages (from matplotlib) (0.10.0)\n",
+      "Requirement already satisfied: python-dateutil>=2.1 in /home/pi/.local/lib/python3.7/site-packages (from matplotlib) (2.8.1)\n",
+      "Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.3 in /home/pi/.local/lib/python3.7/site-packages (from matplotlib) (2.4.7)\n",
+      "Requirement already satisfied: google-auth<2,>=1.6.3 in /home/pi/.local/lib/python3.7/site-packages (from tensorboard<2.1.0,>=2.0.0->tensorflow) (1.11.2)\n",
+      "Requirement already satisfied: werkzeug>=0.11.15 in /usr/lib/python3/dist-packages (from tensorboard<2.1.0,>=2.0.0->tensorflow) (0.14.1)\n",
+      "Requirement already satisfied: google-auth-oauthlib<0.5,>=0.4.1 in /home/pi/.local/lib/python3.7/site-packages (from tensorboard<2.1.0,>=2.0.0->tensorflow) (0.4.1)\n",
+      "Requirement already satisfied: markdown>=2.6.8 in /home/pi/.local/lib/python3.7/site-packages (from tensorboard<2.1.0,>=2.0.0->tensorflow) (3.2.1)\n",
+      "Requirement already satisfied: setuptools>=41.0.0 in /home/pi/.local/lib/python3.7/site-packages (from tensorboard<2.1.0,>=2.0.0->tensorflow) (45.2.0)\n",
+      "Requirement already satisfied: requests<3,>=2.21.0 in /usr/lib/python3/dist-packages (from tensorboard<2.1.0,>=2.0.0->tensorflow) (2.21.0)\n",
+      "Requirement already satisfied: h5py in /home/pi/.local/lib/python3.7/site-packages (from keras-applications>=1.0.8->tensorflow) (2.10.0)\n",
+      "Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/pi/.local/lib/python3.7/site-packages (from google-auth<2,>=1.6.3->tensorboard<2.1.0,>=2.0.0->tensorflow) (0.2.8)\n",
+      "Requirement already satisfied: cachetools<5.0,>=2.0.0 in /home/pi/.local/lib/python3.7/site-packages (from google-auth<2,>=1.6.3->tensorboard<2.1.0,>=2.0.0->tensorflow) (4.0.0)\n",
+      "Requirement already satisfied: rsa<4.1,>=3.1.4 in /home/pi/.local/lib/python3.7/site-packages (from google-auth<2,>=1.6.3->tensorboard<2.1.0,>=2.0.0->tensorflow) (4.0)\n",
+      "Requirement already satisfied: requests-oauthlib>=0.7.0 in /usr/lib/python3/dist-packages (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.1.0,>=2.0.0->tensorflow) (1.0.0)\n",
+      "Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/pi/.local/lib/python3.7/site-packages (from pyasn1-modules>=0.2.1->google-auth<2,>=1.6.3->tensorboard<2.1.0,>=2.0.0->tensorflow) (0.4.8)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip3 install dlr numpy pillow tensorflow matplotlib"
```

Comment:
dlr for ARM is not available using pip from pypi. We should instead pip install from https://neo-ai-dlr-release.s3-us-west-2.amazonaws.com/v1.3.0/pi-armv7l-raspbian4.14.71-glibc2_24-libstdcpp3_4/dlr-1.3.0-py3-none-any.whl

Reply:
Resolved, but see https://github.com/neo-ai/neo-ai-dlr/pull/254#issuecomment-715585950